client/asset/dcr/spv.go

```diff
@@ -240,7 +259,45 @@ func createSPVWallet(pw, seed []byte, dataDir string, extIdx, intIdx uint32, cha
 	return nil
 }
 
-func (w *spvWallet) Reconfigure(ctx context.Context, cfg *asset.WalletConfig, net dex.Network, currentAddress, depositAccount string) (restart bool, err error) {
+// Accounts returns the names of the accounts for use by the exchange wallet.
+func (w *spvWallet) Accounts() XCWalletAccounts {
+	accounts := XCWalletAccounts{
+		PrimaryAccount: defaultAcctName,
+	}
+	if w.hasMixedAccount.Load() {
+		accounts.PrimaryAccount = mixedAccountName
+		accounts.UnmixedAccount = defaultAcctName // the default acct is the unmixed account
+		accounts.TradingAccount = tradingAccount
+	}
+	return accounts
+}
+
+func (w *spvWallet) Reconfigure(ctx context.Context, cfg *asset.WalletConfig, net dex.Network, currentAddress string) (restart bool, err error) {
+	spvCfg, chainParams, err := loadSPVConfig(cfg.Settings, net)
+	if err != nil {
+		return false, fmt.Errorf("error parsing config: %w", err)
+	}
+
+	if chainParams.Net != w.chainParams.Net {
+		return false, errors.New("cannot reconfigure to use different network")
+	}
+
+	startMixer := spvCfg.MixFunds
+	if w.IsMixing() {
+		_ = w.StopFundsMixer() // would only error if mixing wasn't active
+		startMixer = true
+	}
+
+	w.cfgV.Swap(spvCfg)
+
+	if startMixer {
```

Comment:
What if the new config has `MixFunds == false`?

Reply:
Right. I guess that should stop the mixer even if it was running. Will fix.